bigtop-packages/src/common/bigtop-select/distro-select

```diff
@@ -68,7 +68,10 @@ leaves = {
            "flink-historyserver" : "flink",
            "spark-client" : "spark",
            "spark-thriftserver" : "spark",
-           "spark-historyserver" : "spark"
+           "spark-historyserver" : "spark",
+           "ranger-admin" : "ranger-admin",
+           "ranger-usersync" : "ranger-usersync",
+           "ranger-tagsync" : "ranger-tagsync"
```

Comment:
We do not need ranger-kms here?

Reply:
Hi @iwasakims 
Like hdp stack, we can take Ranger Kms as a service rather than a component of ranger service
So i think we can add ranger-kms  service in  other jira ticket

Reply:
Hi @iwasakims 
Like hdp stack, we can take Ranger Kms as a service rather than a component of ranger service
So i think we can add ranger-kms  service in  other jira ticket